### PR TITLE
Portfolio Website: mobile version

### PR DESCRIPTION
**Pull request by HossainAraf**
This project is assigned by Microverse. The following technologies were implemented here:

1. Git & Github
2. Linters: Hint, Stylelint
3. HTML
4. CSS, CSS-grid, Flex-box

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+     <!-- Work -->
+    <section id="work">
```
Comment (V-Blaze) [inline index.html:45]: - [ ] This work `section tag` doesn't have a corresponding closing tag, I suggest you check the closing point of your work section and add the appropriate closing tag. :+1: :smile:

File: index.html [hunk 2]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+          <div> <img src="img/Portfolio-1.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <!-- <p>&#x2022;</p>
+                <P> Full Stack Dev</P>
+                <p>&#x2022;</p>
+                <p>2023</p> -->
```
Comment (V-Blaze) [inline index.html]: - [ ] Following the best practices, you are not supposed to have unused code as a comment in your codebase, it makes your codebase look unprofessional. :+1:

File: index.html [hunk 3]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+          <div> <img src="img/Portfolio-2.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Multi-Post Stories</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <!-- <p>&#x2022;</p>
+                <P> Full Stack Dev</P>
+                <p>&#x2022;</p>
+                <p>2023</p> -->
```
Comment (V-Blaze) [inline index.html]: - [ ] As I suggested above, Following the best practices, you are not supposed to have unused code as a comment in your codebase, it makes your codebase look unprofessional. :+1:

File: index.html [hunk 4]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <button class="language-btn"> <p>html</p></button>
+          <button class="language-btn"><p>css</p></button>
+          <button class="language-btn"><p>Javascript</p></button>
+        </div>
```
Comment (V-Blaze) [inline index.html]: - [ ] These list of technologies are meant to be inside a `ul` tag and them being a `li` item, I suggest you update these. :+1: 
- [ ] Also looking at the given Figma design, your text font tends to be smaller than the given design, also your design has a border that is not in the given design, I suggest you update this accordingly. :+1: :100: 

- - Your Current Design:
Screenshot from 2023-06-07 09-16-40

- - Figma design:
Screenshot from 2023-06-07 09-16-27

- [ ] I also suggest you take a look at the margin top and bottom, the given design tends to have more margin than your current implementation. :+1: :smile: 

**Remember to implement this change in all the technology lists in each project card**

File: index.html [hunk 5]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+        <button class="see-project-btn">See Project</button>
+      </div>
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-2.png" alt=""></div>
```
Comment (V-Blaze) [inline index.html]: - [ ] Following the best practices, all image tags must have a valid alt attribute, which makes the images easy to identify by a screen reader, I'll suggest you go through your project list and add the correct text in the image for all attributes. :+1: :smile:

File: index.html [hunk 6]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <button class="language-btn"> <p>html</p></button>
+          <button class="language-btn"><p>css</p></button>
+          <button class="language-btn"><p>Javascript</p></button>
+        </div>
+        <button class="see-project-btn">See Project</button>
```
Comment (V-Blaze) [inline index.html:72]: - [ ] You did great using the button tag here. :+1: But I think there are still some styling issues you need to update here. 
- [ ] Currently, the background color of your button is not the same as the one in Figma, I'll suggest you get the correct color and update this as replicating the given design is one of the requirements for this milestone. :+1: 
- [ ] Also your buttons do not have the interactive properties implemented, You are meant to implement the `hover and pressed` state of your button with the correct background color and text color. I'll suggest you update this too. :+1: :100: 

- - HOVER AND PRESSED STATE FROM DESIGN:
Screenshot from 2023-06-07 09-16-03

File: index.html [hunk 7]
```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+      <div class="socail-media">
+        <a class="social-media-connect" class="flex" href="#">LET'S CONNECT</a>
+        <ul class="social-media-icon">
+          <li><a href="#" title="Twitter"><img src="img/twitter.svg" alt="twitter logo"></a></li>
+          <li><a href="#" title="Linkedin"><img src="img/Linkedin.svg" alt="Linkedin logo"></a></li>
+          <li><a href="#" title="Medium"><img src="img/medium.svg" alt="Medium logo"></a></li>
+          <li><a href="#" title="Github"><img src="img/github.svg" alt="Github logo"></a></li>
+          <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
+        </ul>
+      <button id="resume-btn">Get my resume</button>
```
Comment (V-Blaze) [inline index.html:174]: - [ ] Just as I suggested above in the `See Project` buttons, I'll suggest you update the button styling:` Active, Hover and Pressed` here too. :+1: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @HossainAraf  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Github flow✔
-  Readme looks professional👋 
-  Linters setup and passing✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HossainAraf** [comment]
Note that, 

- I did not use the hover, active and pressed action for tags (HTML, CSS, Javascript). Rather I removed the buttons, as there is no button at the figma design. However, I changed other buttons (See project and Get my resume) used in the project.
- And amended everything as per review comments.

File: index.html [hunk 8]
```diff
@@ -40,6 +41,161 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
...
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <ul class="flex">
+            <li class="language-tag flex"><p>HTML</p></li>
+            <li class="language-tag flex"><p>CSS</p></li>
+            <li class="language-tag flex"><p>Javascript</p></li>
+          </ul>
+        </div>
+        <button class="see-project-btn">See Project</button>
```
Comment (shahadat3669) [inline index.html:72]: 👍 Great job on the implementation! However, there are some improvements needed for the interactive properties of your buttons to align with the design. Here are the suggestions:

- [ ] Hover effect: In the design, the hover color for the button is specified as `#6070FF`, but in your code, you used `#24245c`. Please update the hover color to match the design. Additionally, the design includes a box shadow effect on the buttons, which is currently missing in your implementation. Consider adding the appropriate box shadow styles to match the design.

- [ ] Active effect: According to the design, the active stage color should be `#2230D2`, but you currently have `#0505b3`. Please update the active color to match the design specifications.

By following the design guidelines and incorporating the correct colors and box-shadow styles, you'll ensure that your buttons are visually consistent and aligned with the intended design. Keep up the great work! 👍🎨

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @HossainAraf , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔ Correct Github flow
✔ Linters setup and passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @HossainAraf , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**